internal-channels/README.md

```diff
@@ -0,0 +1,4 @@
+# internal-channels
+This directory is maintained by ART and consumed by OTA such as promoting releases to various channels. See [stabilization-changes.py](hack/stabilization-changes.py).
```

Comment:
ART maintains the `candidate.yaml` file, but the stabilization bot maintains the other internal channels.  Not sure it's worth explaining "who maintains?" in the README though, because that's all in Git history.  Maybe explain that these are channels used by the stabilization script to populate the version-specific public channels, but where don't (yet ;)) have enough support for making these channels public themselves?

Reply:
Updated. Let me know if it does the right thing.